_posts/2024-01-17-devops-and-flutterflow.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: postv2
+font: serif
+title: "Role of FlutterFlow in Modern App Development: Insights for DevOps Teams"
+description: "FlutterFlow is a visual editor for building Flutter apps. It is a good starting point to move github and use DevOps practices to build out a full app"
+date: 2023-10-17
+highlight: monokai
+image: img/devopsflutterflow/cover.png
+author: Matthew Carter
+tags: [Trust policy, GitHub, GitHub Environments, Security, OIDC]
```

Comment:
Product Dev?

Reply:
Ill make a note to change these - ripped the template from another post...